### PR TITLE
Don't show meter toast if 'showToast' is false

### PR DESCRIPTION
**Pull request by elijahsoria**
Updates consume_ in EntitlementsManager so that we only show the meter toast if the field 'showToast' in the metering entitlement response's details is true or not provided. Adds tests to check the various cases.

### CODE REVIEW

**Review by ChrisAntaki — CHANGES_REQUESTED** [review]
Looks good overall! I just requested an additional test to cover a line

File: src/runtime/entitlements-manager.js [hunk 1]
```diff
@@ -479,17 +479,44 @@ export class EntitlementsManager {
...
+   * Gets the `showToast` value (or null/undefined if unavailable) from
+   * the Google metering entitlement details in the input entitlements.
+   * @param {!Entitlements} entitlements
+   * @return {boolean|undefined}
+   * @private
+   */
+  getShowToastFromEntitlements_(entitlements) {
+    const entitlement = entitlements.getEntitlementForThis();
+    if (!entitlement || entitlement.source !== GOOGLE_METERING_SOURCE) {
+      return;
```
[the remark this answers is not in the document]
Comment (elijahsoria) [reply src/runtime/entitlements-manager.js:509]: Done.

**Review by elijahsoria — COMMENTED** [review]
Added test which should cover the missing line.

**Review by ChrisAntaki — APPROVED** [review]
Perfect, thanks!

**Pull request merged**